Sources/TokamakCore/Views/Buttons/Button.swift

```diff
@@ -48,7 +48,7 @@ public struct Button<Label>: View where Label: View {
   }
 }
 
-public struct _Button<Label>: PrimitiveView where Label: View {
+public struct _Button<Label>: _PrimitiveView where Label: View {
```

Comment:
`PrimitiveView` is not meant to be re-exported by any renderer modules, making that explicit here with underscore naming.